docs/getting-started/cli.md

```diff
@@ -0,0 +1,198 @@
+---
+slug: cli
+title: CLI-only workflow (Pro)
+hide_title: true
+description: Learn how to set up Semgrep, scan your first project for security issues, and view your findings in the CLI.
+tags:
+  - quickstart
+  - cli
+---
+
+import MoreHelp from "/src/components/MoreHelp";
+import Install from "/src/components/procedure/_install-cli.mdx";
+import Login from "/src/components/procedure/_login-activate.mdx";
+
+# CLI-only workflow with Semgrep's Pro Engine
+
+Learn how to set up Semgrep, scan your project for security issues using Semgrep's Pro Engine, and view your findings in the CLI.
+
+:::info
+For scans in your local development environment, `semgrep scan` is the preferred method to scan with Semgrep, especially for writing and testing custom rules.
+:::
+
+## Prerequisites
+
+Before proceeding:
+
+* See [Prerequisites](/prerequisites/) to ensure that your machine meets Semgrep's requirements.
+* Ensure that you have and are logged into your [Semgrep Account](https://semgrep.dev/login).
+* Ensure that you've enabled the **Pro Engine** <i class="fa-solid fa-toggle-large-on"></i> toggle on Semgrep Cloud Platform's [Settings](https://semgrep.dev/orgs/-/settings) page.
+
+## Set up Semgrep
+
+<Install />
+
+## Log into your Semgrep account
+
+<Login />
+
+You must log in to your Semgrep account before proceeding. Otherwise, you can't use `semgrep ci`.
+
+## Scan your project
+
+Navigate to the root of your repository, and run your first scan. The specific command you use depends on how you want to view the results.
+
+To view the results in the CLI:
+
+```console
+semgrep ci
+```
+
+To export the results to a plain text file:
+
+```console
+semgrep ci --text --output=semgrep.txt
+```
+
+To export the results to a SARIF file:
+
+```console
+semgrep ci --sarif --output=semgrep.sarif
+```
+
+To export the results to a JSON file:
+
+```console
+semgrep ci --json --output=semgrep.json
+```
+
+### Scan your project with a specific ruleset
+
+You can scan your project using `--config auto` to automatically obtain rules tailored to your project:
+
+```console
+semgrep scan --config auto
+```
+
+:::info
+Semgrep collects pseudonymous metrics when you use rules from the Registry.
```

Comment:
Same change as my suggestion in the other doc - we should include how to turn metrics off